crash-recovery/README.md

```diff
@@ -0,0 +1,78 @@
+```
+shortname: ?/CRASH-RECOVERY
+name: Restore system state after crash
+type: standard
+status: raw
+editor: Vanshdeep Singh <vanshdeep@bigchaindb.com>
+```
+
+## Abstract
+Unexpect crashes are bound to occur when BigchainDB is deployed in production which could leave the system in an inconsistant state. Hence it is necessary that an appropriate methodology should be developed to restore the system to a consistant state.
+
+
+## Motivation
+For BigchainDB, a crash which can cause the state to be inconsistent could leave the system inoperable. The 'achilles heel' for BigchainDB is its `commit` function during which a new block with its transactions is written to the disk. Furthermore, UTXO merkle tree is updated based on these new transactions. An overview of what happens during `commit` is given below,
```

Comment:
Put a link on UTXO for people unfamiliar with what it means, so they can find a definition.